packages/react-components/react-search/stories/SearchBox/SearchBoxAppearance.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { SearchBox } from '@fluentui/react-search';
+import { Field, makeStyles, mergeClasses, shorthands, tokens } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: '490px',
+  },
+  filledLighter: {
+    backgroundColor: tokens.colorNeutralBackgroundInverted,
+  },
+  filledLighterLabel: {
+    color: tokens.colorNeutralForegroundInverted2,
+  },
+  filledDarker: {
+    backgroundColor: tokens.colorNeutralBackgroundInverted,
+  },
+  filledDarkerLabel: {
+    color: tokens.colorNeutralForegroundInverted2,
+  },
+  fieldWrapper: {
+    ...shorthands.padding(tokens.spacingVerticalMNudge, tokens.spacingHorizontalMNudge),
+  },
+});
+
+export const Appearance = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.base}>
+      <div className={styles.fieldWrapper}>
+        <Field label="Outline appearance (default)">
+          <SearchBox appearance="outline" />
+        </Field>
+      </div>
+
+      <div className={styles.fieldWrapper}>
+        <Field label="Underline appearance">
+          <SearchBox appearance="underline" />
+        </Field>
+      </div>
+
+      <div className={mergeClasses(styles.fieldWrapper, styles.filledLighter)}>
+        <Field label={{ children: 'Filled lighter appearance', className: styles.filledLighterLabel }}>
+          <SearchBox appearance="filled-lighter" />
+        </Field>
+      </div>
```

Comment:
You can remove the wrapping div and do the following:
```tsx
        <Field className={mergeClasses(styles.fieldWrapper, styles.filledLighter)} label={{ children: 'Filled lighter appearance', className: styles.filledLighterLabel }}>
          <SearchBox appearance="filled-lighter" />
        </Field>
```

same for the other fields